libs/utils/energy.py

```diff
@@ -105,6 +121,8 @@ def getInstance(target, conf, force=False, res_dir=None):
             EnergyMeter._meter = HWMon(target, emeter, res_dir)
         elif emeter['instrument'] == 'aep':
             EnergyMeter._meter = AEP(target, emeter['conf'], res_dir)
+        elif emeter['instrument'] == 'iiocapture':
```

Comment:
I would propose to use "Acme" as instrument name, since iio-capture is a more generic tool which we do not currently support.